scipy/integrate/_py/base.py

```diff
@@ -61,12 +61,15 @@ class OdeSolver(object):
         Current time.
     y : ndarray
         Current state.
+    t_old : float
+        Time of the last successful step. None if no steps were made yet.
```

Comment:
Now I think that it sounds strange. "End time of the last successful step" might as well be `self.t`. I suggest simply `Previous time`, and maybe change `t_old` to `t_prev` everywhere.

By the way, what is the special purpose of `t_old` as an attribute?